lib/install/data/data.go

```diff
@@ -70,6 +70,8 @@ type Data struct {
 
 	Force bool
 	UseRP bool
+
+	ScratchSize string
```

Comment:
Comes in as a string from the user on the command line so that they can use G/K/T/P to specify units. Gets parsed into an int (in kilobytes, because that's what vSphere takes) before it's stored in the `VirtualContainerHostConfigSpec`